src/tools/getFileWithAnalysisTool.ts

```diff
@@ -1,9 +1,22 @@
+import { generalOrganizationMistakes, generalRepositoryMistakes } from '../rules.js';
 import { CodacyTool, fileSchema, toolNames } from '../schemas.js';
 
+const rules = `
+ If a file is ignored, it is not analyzed by Codacy. 
+ Using this tool you'll get results for five metrics: Grade, Issues, Duplication, Complexity and Coverage.
+
+ Common use cases: 
+ - When the user asks for the analysis of a specific file.
```

Comment:
```suggestion
 - When the user asks for the analysis of a specific file.
```
what it the user prompt is: "analyze this file" do you think it will use this tool instead of cli?